### DISCUSSION TITLE
after upgrade angular 8 to 9, my old package is not working correctly

### DISCUSSION THREAD

**Discussion by MdMirajAnsari** [community/community#34559] [Other Feature Feedback, Questions, & Ideas]
after upgrade angular 8 to 9, my old package is not working correctly

**Comment by jfullstackdev** [comment]
maybe this will help 

https://stackoverflow.com/questions/60850311/upgrade-angular-version-8-9

**Comment by igor-h** [comment]
Hello @mirajhad 

Try to follow those steps:

https://update.angular.io/?l=3&v=8.0-9.1


![image](https://user-images.githubusercontent.com/39650812/194938040-da0cde7a-4960-4a53-937e-bad992e75f65.png)

_If this was helpful mark my comment as Resolved._

Thanks, Igor. 👍🏽

**Reply by MdMirajAnsari** [reply]
I have tried that one also but didn't work for me.

**Reply by igor-h** [reply]
@mirajhad in your project folder, remove **node_modules**,
and run **npm install**  
And see if is working now.
just in case check... in package.json you have updated angular version also. 👍🏽
